docker-compose/monitor/Makefile

```diff
@@ -29,3 +34,15 @@ clean-all: clean-jaeger
 	docker rmi -f otel/opentelemetry-collector-contrib:latest ; \
 	docker rmi -f prom/prometheus:latest ; \
 	docker rmi -f grafana/grafana:latest
+
+.PHONY: build-v2
+build-v2: clean-jaeger
+	cd ../../ && make build-jaeger-linux
```

Comment:
instead of repeating, can you parameterize with BINARY var, similar to how scripts/build-all-in-one-image.sh does it?

Reply:
Do the below two commands do the same thing ?
```Makefile
make build-v2
```
```Makefile 
make build-jaeger GOOS=linux GOARCH="${arch}" 
```

Reply:
Should I parameterize the script or the Makefile ?

Reply:
> Do the below two commands do the same thing ?

no. the former builds a container, the latter only the binary.

> Should I parameterize the script or the Makefile ?

Reply:
Done. Do you want me to parameterize the version also in scripts/spm-integration-test.sh ?

Reply:
looks ok as is.